examples/workflow/pygrb/pygrb.ini

```diff
@@ -158,35 +159,45 @@ channel-name = ${workflow|l1-channel-name}
 random-sort =
 
 [inspiral]
-; inspiral analysis parameters -- added to all inspiral jobs
-pad-data = 8
-segment-length = 1048576
-number-of-segments = 15
+; coh_PTF_inspiral analysis parameters -- added to all inspiral jobs
+ligo-calibrated-data = real_8
+low-template-freq = 40.0
+highpass-frequency = 35
+snr-threshold = 6.
+trig-time-window = 1
+low-template-freq = 38
+low-filter-freq = 40
+high-filter-freq = 1000
+strain-data =
 sample-rate = 4096
+segment-duration = 256
+;number-of-segments = 5
+block-duration = 5248
+pad-data = 8
+do-trace-snr =
+;do-null-stream
+do-bank-veto =
+do-auto-veto =
+do-chi-square =
+num-auto-chisq-points = 40
+auto-veto-time-step = 0.001
+bank-veto-templates = bank_veto_bank.xml
```

Comment:
I think this should either be handled by pegasus (in which case it would not be specified explicitly here) .... Alex and I once considered implementing something like bank-veto-templates = {pegasusfile://path/fo/file} or something similar to let the code know that it should be creating a pegasus file object for this input (if it doesn't already exist) and transferring it correctly ..... we just never got around to thinking through how to do this properly and implement it.

In the meantime, this should probably be a full path